tests/lib/rules/consistent-return.js

```diff
@@ -255,6 +316,7 @@ ruleTester.run("consistent-return", rule, {
                     messageId: "missingReturn",
                     data: { name: "program" },
                     type: "Program",
+                    line: 1,
                     column: 1
```

Comment:
Every other invalid test case asserts end location information, but we don't have an end location for `Program` reports. Do you think it's worth adding these to demonstrate to future readers that this omission is intentional?

```suggestion
                    line: 1,
                    column: 1,
                    endLine: void 0,
                    endColumn: void 0 
```

Reply:
Makes sense, I will add these.